modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/linear-segregated/steady-transient-compare/common-blocks.i

```diff
@@ -127,6 +127,7 @@ alpha_b = 1e-4
   [h_advection]
     type = LinearFVEnergyAdvection
     variable = T
+    advected_quantity = temperature
```

Comment:
Just a note for the future: at some point we can try to detect this automatically.

Reply:
The inital plan was defining special variables right?

Reply:
this isnt great (advected is cp T not T) but in the optics of using Physics instead anyway I dont mind

Reply:
this isnt great (advected is cp T not T) but in the optics of using Physics instead anyway I dont mind